### PR TITLE
add netlify text and icon to the footer

### PR DESCRIPTION
**Pull request by cherylli**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |

#### Have you updated the CHANGELOG.md file? If not, please do it.
yes

#### What is this change?
Added the `This site is powered by` text and the Netlify Icon
This is in preparation for the application for the Netlify Open Source plan.
https://www.netlify.com/legal/open-source-policy/

Let me know if there's any suggestions for the text/logo placement etc

image

image

#### Were there any complications while making this change?
no

#### How to replicate the issue?
n/a

#### If necessary, please describe how to test the new feature or fix.
n/a

#### When should this be merged?
after review

### CODE REVIEW

File: components/layout/Footer/index.js [hunk 1]
```diff
@@ -50,11 +50,19 @@ export default function Footer() {
...
         <S.Copyright>
           © Web Dev Path {new Date().getFullYear()}. All rights reserved.
         </S.Copyright>
-      </Container>
+        <S.Netlify>
+          This site is powered by
+          <Image
+            src='images/svg/logo-netlify-small-fullcolor-darkmode.svg'
+            width={50}
+            height={50}
```
Comment (Satoshi-Sh) [inline components/layout/Footer/index.js:62]: We need to add an alt for the accessibility.

**Review by Satoshi-Sh — COMMENTED** [review]
Looks nice. We just need to add an alt. 

I didn't know about the Netlify Open Source plan. Will it be free if they accept?

File: components/layout/Footer/index.js (hunk 1 above)
Comment (cherylli) [reply components/layout/Footer/index.js:62]: thanks forgotten about it 
and yes we will get a free pro plan I believe, if they accept the application

Comment (Satoshi-Sh) [reply components/layout/Footer/index.js:62]: Welcome. 

I hope they accept the application.

**Review by Satoshi-Sh — APPROVED** [review]

**Review by mariana-caldas — APPROVED** [review]
This is great, @cherylli ! Thanks! 🚀

**Pull request merged**